api.d.ts

```diff
@@ -38,9 +38,9 @@ declare const jr_is_readonly: (id: ElementId, customHandler?: CustomErrorHandler
 declare const jr_set_disabled: (id: ElementId, value?: boolean, customHandler?: CustomErrorHandler) => void;
 declare const jr_is_disabled: (id: ElementId, customHandler?: CustomErrorHandler) => boolean;
 declare const jr_set_background_color: (id: ElementId, color: string, customHandler?: CustomErrorHandler) => void;
-declare const jr_set_subtable_value: (subtableViewName: string, rowId: number, columnName: string, value: any, customHandler?: CustomErrorHandler) => void;
+declare const jr_set_subtable_value: (subtableViewName: string, rowId: number | "sum", columnName: string, value: any, customHandler?: CustomErrorHandler) => void;
```

Comment:
This "feature" is not documented as we do not want users to set the sum row manually.
To update the sum row, we recommend using jr_sum_subtable_column as it does the calculation for you.

Reply:
Resolved in https://github.com/jobrouter/javascript-api/pull/3/commits/6bbfda3aa249cb15377687df87469f2fd82897cd